From ba354c92bd56b31dfc9231847c78934d908dd9bf Mon Sep 17 00:00:00 2001
From: Michael Abood <person4265@gmail.com>
Date: Mon, 8 Nov 2021 11:33:36 -0500
Subject: [PATCH 194/210] drm/panel: panel-samsung-nt51017-b4p096wx5vp09: Fix
 warning, add sleep

linux-mdss-dsi-panel-driver-generator generates an empty function
with an unused variable. This removes that line. Additionally, it
adds a sleep that corresponds with samsung,mdss-power-on-reset-delay-us
in the downstream dtb.

Signed-off-by: Michael Abood <person4265@gmail.com>
---
 .../msm8916-generated/panel-samsung-nt51017-b4p096wx5vp09.c   | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-nt51017-b4p096wx5vp09.c b/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-nt51017-b4p096wx5vp09.c
index c2a554bb1e4..2f308dcf889 100644
--- a/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-nt51017-b4p096wx5vp09.c
+++ b/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-nt51017-b4p096wx5vp09.c
@@ -53,8 +53,6 @@ static int nt51017_on(struct nt51017 *ctx)
 
 static int nt51017_off(struct nt51017 *ctx)
 {
-	struct mipi_dsi_device *dsi = ctx->dsi;
-
 	return 0;
 }
 
@@ -73,6 +71,8 @@ static int nt51017_prepare(struct drm_panel *panel)
 		return ret;
 	}
 
+	msleep(30);
+
 	ret = nt51017_on(ctx);
 	if (ret < 0) {
 		dev_err(dev, "Failed to initialize panel: %d\n", ret);
-- 
2.34.1

